opds2_import.py

```diff
@@ -71,7 +71,16 @@ def parse_manifest(self, manifest):
         """
         result = None
 
-        if isinstance(manifest, str):
+        if isinstance(manifest, bytes):
+            try:
+                input_stream = StringIO(manifest.decode("utf-8"))
+                parser = self._manifest_parser_factory.create()
+                result = parser.parse_stream(input_stream)
+            except BaseError:
+                logging.exception("Failed to parse the RWPM-like manifest")
```

Comment:
This string is repeated three times within this function. It would be nice to use a constant here (and anywhere else it might appear).